data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.gateway/1.0/corda.p2p.gateway.json

```diff
@@ -5,24 +5,35 @@
   "description": "Configuration schema for the P2P gateway.",
   "type": "object",
   "properties": {
-    "hostAddress": {
-      "description": "The host name or IP address where the HTTP server will bind.",
-      "type": "string",
-      "format": "hostname",
-      "default": "0.0.0.0"
-    },
-    "hostPort": {
-      "description": "The HTTP port.",
-      "type": "integer",
-      "default": 8080,
-      "minimum": 0,
-      "maximum": 65535
-    },
-    "urlPath": {
-      "description": "The url path the gateway server will be listening to.",
-      "type": "string",
-      "default": "/",
-      "pattern": "^\\/[\\/.a-zA-Z0-9-]*$"
+    "serversConfiguration": {
+      "type": "array",
+      "minItems": 1,
+      "description": "The HTTP servers. The pair hostAddress and hostPort must be unique for each server.",
+      "default": [{"hostPort": 8080}],
```

Comment:
maybe it doesn't matter depending on how default takes effect (since if it's at least one entry, I would assume defaults for that type apply), but should we have all fields defined at the default at this level anyway? e.g.
```
"default": [{"hostAddress": "0.0.0.0", "hostPort": 8080, "urlPath": "/"}]
``` 
I think it would be a nice way to document the full default.

Reply:
Annoyingly, if we put the default just in the array, it will not default any additional element in the group so I had to put them twice :(